GoogleForCreators/web-stories-wp
# Add uppercase toggle to the Text Style panel

## Summary
Adds uppercase Toggle to the text styles panel. Can be used for inline styles.


## Relevant Technical Choices



## To-do

- [x] ~Keyboard shortcut (TBD)~ Not needed.



## User-facing changes
The text styles panel now has uppercase as an option, next to Underline.

![Screenshot 2021-06-11 at 15 21 49](https://user-images.githubusercontent.com/3294597/121685581-64497780-cac0-11eb-9e83-2f7e461fc053.png)



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add a text element
2. Verify you can select part of it and mark it uppercase
3. Verify you can select all the text and mark it uppercase
4. Verify you can combine different styles with uppercase (e.g. one word is uppercase but only part of it is bold)
5. Verify it also work in the front-end.
6. Verify that uppercase can be added to multiple elements together.

## Checklist




---



Fixes #6015
